File: _persist/temp/venda/CSharp/Transimposto.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSIMPOSTO")]
    public class Transimposto : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.tfKey)]
        public string Id_Transacao { get; set; }
        [Campo("NR_ITEM", CampoTipo.tfKey)]
        public int Nr_Item { get; set; }
        [Campo("CD_IMPOSTO", CampoTipo.tfKey)]
        public int Cd_Imposto { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.tfReq)]
        public double Pr_Aliquota { get; set; }
        [Campo("VL_BASECALCULO", CampoTipo.tfReq)]
        public double Vl_Basecalculo { get; set; }
        [Campo("PR_BASECALCULO", CampoTipo.tfReq)]
        public double Pr_Basecalculo { get; set; }
        [Campo("PR_REDBASECALCULO", CampoTipo.tfReq)]
        public double Pr_Redbasecalculo { get; set; }
        [Campo("VL_IMPOSTO", CampoTipo.tfReq)]
        public double Vl_Imposto { get; set; }
        [Campo("VL_OUTRO", CampoTipo.tfReq)]
        public double Vl_Outro { get; set; }
        [Campo("VL_ISENTO", CampoTipo.tfReq)]
        public double Vl_Isento { get; set; }
        [Campo("CD_CST", CampoTipo.tfReq)]
        public string Cd_Cst { get; set; }
        [Campo("CD_CSOSN", CampoTipo.tfNul)]
        public string Cd_Csosn { get; set; }
    }
}

File: _persist/temp/venda/Java/Transimposto.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("TRANSIMPOSTO")
public class Transimposto extends CollectionItem
{
    @Campo("ID_TRANSACAO", CampoTipo.tfKey)
    public String Id_Transacao;
    @Campo("NR_ITEM", CampoTipo.tfKey)
    public Integer Nr_Item;
    @Campo("CD_IMPOSTO", CampoTipo.tfKey)
    public Integer Cd_Imposto;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("PR_ALIQUOTA", CampoTipo.tfReq)
    public Float Pr_Aliquota;
    @Campo("VL_BASECALCULO", CampoTipo.tfReq)
    public Float Vl_Basecalculo;
    @Campo("PR_BASECALCULO", CampoTipo.tfReq)
    public Float Pr_Basecalculo;
    @Campo("PR_REDBASECALCULO", CampoTipo.tfReq)
    public Float Pr_Redbasecalculo;
    @Campo("VL_IMPOSTO", CampoTipo.tfReq)
    public Float Vl_Imposto;
    @Campo("VL_OUTRO", CampoTipo.tfReq)
    public Float Vl_Outro;
    @Campo("VL_ISENTO", CampoTipo.tfReq)
    public Float Vl_Isento;
    @Campo("CD_CST", CampoTipo.tfReq)
    public String Cd_Cst;
    @Campo("CD_CSOSN", CampoTipo.tfNul)
    public String Cd_Csosn;
}